planner/core/testdata/integration_serial_suite_in.json

```diff
@@ -279,6 +279,7 @@
       "desc format = 'brief' select /*+hash_agg()*/ sum(id) from (select value, id from t where id > value group by id, value)A group by value /*the exchange should have only one partition column: test.t.value*/",
       "desc format = 'brief' select /*+hash_agg()*/ sum(B.value) from t as B where B.id+1 > (select count(*) from t where t.id= B.id and t.value=B.value) group by B.id /*the exchange should have only one partition column: test.t.id*/",
       "desc format = 'brief' select count(distinct value) from t",
+      "desc format = 'brief' select count(distinct x ) from (select count(distinct value) x from t) t",
```

Comment:
would we add a test case : select count(distinct x), avg(y) ? becase avg would add a proj.

Reply:
the test case below this one is exactly what you mentioned

Reply:
ok